### PR TITLE
fix: Tabs component: underlying HTML

### PR DESCRIPTION
**Pull request by itmilos**
## Related Issue
Closes SAP/fundamental-styles#105

## Description
Changed anchor tags to button tags

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [na] All values are in `rem`
- [na] Text elements follow the truncation rules
- [na] hover state of the element follow design spec
- [na ] focus state of the element follow design spec
- [na] active state of the element follow design spec
- [na] selected state of the element follow design spec
- [na] selected hover state of the element follow design spec
- [na] pressed state of the element follow design spec
- [na] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [na] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [na] BEM naming convention is used
- [na] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [na] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [na] `fd-reset()` mixin is applied to all elements
- [na] Variables are used, if some value is used more than twice.
- [na] Checked if current components can be reused, instead of having new code.
3. Testing
- [na] tested Storybook examples with "CSS Resources" `normalize` option 
- [na] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [ ] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [ ] Storybook documentation has been created/updated
- [ ] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

**Review by droshev — CHANGES_REQUESTED** [review]
You need to update the tests.
Can you mark and document this as a breaking change?
Also, can you check in other places if the tabs are used and update the markup?

**Comment by itmilos** [comment]
I added **role="presentation"** to **li** element and **role="tab"** to **button** ref. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role

**Comment by itmilos** [comment]
> I added **role="presentation"** to **li** element and **role="tab"** to **button** ref. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role

This approach does not meet the accessibility tests standard

**Comment by itmilos** [comment]
## 0.17.0

#### Tabs - change element  from `a` to `button`

Before:
```
 <li role="tab" class="fd-tabs__item">
                            <a class="fd-tabs__link">
                                <span class="fd-tabs__tag">
                                    Link
                                </span>
                            </a>
                        </li>
...
```

After:
```
 <li role="tab" class="fd-tabs__item">
                            <button class="fd-tabs__link">
                                <span class="fd-tabs__tag">
                                    Link
                                </span>                           
 </button>
                        </li>
...
```

File: stories/notification/notification.stories.js [hunk 1]
```diff
@@ -202,34 +202,31 @@ Notifications can display error alerts by adding the <code>fd-message-strip fd-m
 export const notificationGroup = () => `<div class="fd-notification fd-notification--group">
     <ul class="fd-tabs fd-tabs--l fd-notification--tabs" role="tablist">
         <li role="tab" class="fd-tabs__item" aria-selected="true">
-            <a 
+            <button
```
Comment (InnaAtanasova) [inline stories/notification/notification.stories.js:205]: Buttons are missing tabindex, also missing is the aria-selected

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
The examples are no longer changing the tab panels. 
To reproduce:
- click on a different Tab
- the tab panel content is not updated

**Comment by droshev** [comment]
@itmilos you can move to branch PR

**Pull request closed without merging**